Library/Homebrew/keg.rb

```diff
@@ -339,14 +339,14 @@ def to_formula
   end
 
   def installed_dependents
+    return [] unless linked?
```

Comment:
If I have two versions of _A_ installed, and _B_ depends on _A_, if we just checked the name, I wouldn't be able to uninstall my old version of _A_ that isn't needed anymore. We need a way to differentiate the "current" version of _A_ from the other installed versions, and checking if it's linked seems to be the most appropriate way to do that.